manifests/t/Telegram/Unigram/10.2.3.9268/Telegram.Unigram.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Telegram.Unigram
+PackageVersion: 10.2.3.9268
+Platform:
+- Windows.Universal
+MinimumOSVersion: 10.0.18362.0
+InstallerType: msix
+PackageFamilyName: TelegramFZ-LLC.Unigram_1vfw5zm9jmzqy
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.UI.Xaml.2.8
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 860.)